tools/test.py

```diff
@@ -1554,9 +1554,8 @@ def PrintCrashed(code):
   'gc',
   'internet',
   'pummel',
-  'test-known-issues',
+  'known_issues',
```

Comment:
I'm surprised `known_issues` is in the ignore set. Any objection to removing it?

Reply:
(probably) fine with removing it, but might be best to do in a separate PR for visibility.

Reply:
But if this lands then won't we stop running `known_issues` until the subsequent PR to remove it from this list lands?

Reply:
@richardlau Ah, true -- I'll remove `known_issues` from the list in this PR :)